Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG GO_VERSION=1.17
+
+FROM golang:${GO_VERSION}-alpine as build
+
+RUN apk --no-cache add make ca-certificates 
```

Comment:
are the ca-certificates really necessary for the build?

Reply:
ca-certificate was not needed. Left over from a previous test. Removed.